examples/conformer_energies/conformer_energies.py

```diff
@@ -29,42 +30,59 @@ def compute_conformer_energies_from_file(filename):
 
     n_molecules = len(molecules)
     n_conformers = sum([mol.n_conformers for mol in molecules])
-    print(f'{n_molecules} unique molecule(s) loaded, with {n_conformers} total conformers')
+    print(
+        f"{n_molecules} unique molecule(s) loaded, with {n_conformers} total conformers"
+    )
 
     # Load the openff-1.1.0 force field appropriate for vacuum calculations (without constraints)
```

Comment:
(blocking) Thanks! @mattwthompson's approval stands, I just wanted to add this one to-do because this script is special case. This file is kinda weird since its neither library code nor a notebook example, and the way that it's structured will make this be a "silent behavior change" for the users. So in addition to this change, we should

- [ ] Update the README for this example to state that it now uses 2.0.0
- [ ] Record the change of force field in the `Behavior changed` section of the releasehistory. 
- [ ] (if possible) print which FF is used in the print statement at the end of this script

```suggestion
    # Load the openff-2.0.0 force field appropriate for vacuum calculations (without constraints)
```